v2/apiserver/internal/core/mongodb/projects_store.go

```diff
@@ -120,15 +121,81 @@ func (p *projectsStore) ListSubscribers(
 	event core.Event,
 ) (core.ProjectList, error) {
 	projects := core.ProjectList{}
-	subscriptionMatchCriteria := bson.M{
+	// Finding all Projects that are subscribed to a given Event is rather tricky.
```

Comment:
Wow, accruing advanced mongo knowledge.  Hadn't seen this method.

Reply:
I just learned this trick in the last few days. I'd rather not have done this, but it's the only way I could find to do the "reverse" label match.